### PR TITLE
user model added

### PR DESCRIPTION
**Pull request by harsh-ux**

### CODE REVIEW

**Comment by DarshitJain04** [comment]
@harsh-ux maybe you can update the commit message to something which describes what you have added..like which model

**Comment by harsh-ux** [comment]
> @harsh-ux maybe you can update the commit message to something which describes what you have added..like which model

I think I was told to make only one model that was user model. will make sure to specify things

**Comment by DarshitJain04** [comment]
> > @harsh-ux maybe you can update the commit message to something which describes what you have added..like which model
> 
> I think I was told to make only one model that was user model. will make sure to specify things 

That's not the issue. It's just that it would be easy to track commits for those who don't know about the model you were working on

**Comment by harsh-ux** [comment]
> > > @harsh-ux maybe you can update the commit message to something which describes what you have added..like which model
> > 
> > 
> > I think I was told to make only one model that was user model. will make sure to specify things
> 
> That's not the issue. It's just that it would be easy to track commits for those who don't know about the model you were working on

thanks will keep in mind

File: backend/config/settings.py [hunk 1]
```diff
@@ -89,7 +97,19 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.sqlite3',
+<<<<<<< Updated upstream
```
Comment (manav1403) [inline backend/config/settings.py:100]: remove these merge conflicts

**Pull request closed without merging**